nursery/list-user-account-groups.yml

```diff
@@ -0,0 +1,11 @@
+# generated using capa explorer for IDA Pro
+rule:
+  meta:
+    name: list user account groups
+    namespace: host-interaction/accounts
```

Comment:
can you add a description here that differentiates this rule from `list groups for user account`?

Reply:
```suggestion
    namespace: host-interaction/accounts
    description: enumerates all the groups present on the system/domain
```

maybe?

Reply:
no problem I can see these names getting confusing. love the suggested description.